### ISSUE TITLE
Travis does not like having Rmosek in the Suggests field.

### ISSUE THREAD

**Issue by dcgerard** [dcgerard/vicar#1]
I'm leaving out Rmosek from Suggests until I find out how to get Travis to like it.

**Comment by dcgerard** [comment]
It doesn't work even when I set

``` yaml
- _R_CHECK_FORCE_SUGGESTS_="FALSE"
```

I should look at this again sometime and put Rmosek back into the Suggests field. I would also need to uncomment some code in [mouthwash_uniform_coordinate.R](https://github.com/dcgerard/vicar/blob/master/R/mouthwash_uniform_coordinate.R), specifically the code

``` r
        ## if (requireNamespace(package = "Rmosek", quietly = TRUE)) {
        ##     optmethod <- "mixIP"
        ## } else {
        ##     optmethod <- "mixEM"
        ## }
```

**Comment by dcgerard** [comment]
Solved by f3616f92bbffe9aa2216d6252d2ceb02767bdf74 @dcgerard dcgerard committed 3 hours ago
